### PR TITLE
Update psuedo-selectors.md

### PR DESCRIPTION
**Pull request by MrWadeDotNet**

### CODE REVIEW

File: front-end-notes/psuedo-selectors.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+img: hover {
+ color: #000000;
+}
+```
+
+##### References
+
+<ul>
+<li>https://github.com/nashville-software-school/front-end-curriculum/tree/master/11-css-102</li>
+<li>https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started/Selectors
```
Comment (danielcossins11) [inline front-end-notes/psuedo-selectors.md]: Your missing a closing <li> tag. Also, markdown has it's own syntax for lists. Just use a *

**Comment by rhondae1** [comment]
Can you give a brief description of what a pseudo-selector is and why we use it?

File: front-end-notes/psuedo-selectors.md (hunk 1 above)
Comment (rick8s) [inline front-end-notes/psuedo-selectors.md]: Might be helpful to provide an explanation of what the example code is doing.

File: front-end-notes/psuedo-selectors.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#Psuedo-class selectors
+
+###A CSS pseudo-class is a keyword added to selectors that specifices a speical state of the element to be selected.
+
+##### Example
+```css
+ selector: pseduo-class {
+  property: value;
+  };
```
Comment (sethdorris) [inline front-end-notes/psuedo-selectors.md]: you don't need a semi-colon after the closing bracket on line 9.. what is this .. JSCSS? lol

**Comment by gregkorte** [comment]
Lewis. Can you clean up the Example2 header so I can merge this with the curriculum? Thanks.

**Pull request closed without merging**